main/integration-tests/support/resources/workspace-service-catalogs/workspace-service-catalog.js

```diff
@@ -55,7 +55,7 @@ class WorkspaceServiceCatalog extends Resource {
 
   async cidr(body) {
     const api = `${this.api}/cidr`;
-    const response = await this.doCall(async () => this.axiosClient.put(api, body, {}));
+    const response = await this.doCall(async () => this.axiosClient.post(api, body, {}));
```

Comment:
Making updates to CIDR is actually a `POST` method, not a `PUT` method. ([Link here](https://github.com/awslabs/service-workbench-on-aws/blob/a34c414d3e9e7fd72d6528632b6e63e9ca6391db/addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/environment-sc-controller.js#L89)]